src/mame/video/pacland.cpp

```diff
@@ -379,24 +380,41 @@ uint32_t pacland_state::screen_update(screen_device &screen, bitmap_ind16 &bitma
 	draw_fg(screen, bitmap, cliprect, 0);
 
 	/* draw sprites with regular transparency */
-	draw_sprites(screen, bitmap, cliprect, flip, 1);
+	m_sprite1_bitmap.fill(0, cliprect);
+	draw_sprites(screen, m_sprite1_bitmap, cliprect, flip, 1);
+	for (int y = cliprect.min_y; y <= cliprect.max_y; y++)
```

Comment:
This block can be replaced with copybitmap_trans.

Reply:
Thank you for your advice.
I modified to use copybitmap_trans.